pcsx2/SIO/Memcard/MemoryCardFile.cpp

```diff
@@ -940,15 +932,16 @@ bool FileMcd_CreateNewCard(const std::string_view& name, MemoryCardType type, Me
 
 		if (!FileSystem::CreateDirectoryPath(full_path.c_str(), false))
 		{
-			Host::ReportFormattedErrorAsync("Memory Card Creation Failed", "Failed to create directory '%s'.", full_path.c_str());
+			Host::ReportErrorAsync("Memory Card Creation Failed",
+				fmt::format("Failed to write folder Memory Card:\n{}", full_path.c_str()));
 			return false;
 		}
 
 		// write the superblock
 		auto fp = FileSystem::OpenManagedCFile(Path::Combine(full_path, s_folder_mem_card_id_file).c_str(), "wb");
 		if (!fp)
 		{
-			Host::ReportFormattedErrorAsync("Memory Card Creation Failed", "Failed to write memory card folder superblock '%s'.", full_path.c_str());
+			Host::ReportErrorAsync("Memory Card Creation Failed", fmt::format("Failed to write folder Memory Card superblock:\n{}", full_path.c_str()));
```

Comment:
```suggestion
			Host::ReportErrorAsync("Memory Card Creation Failed", fmt::format("Failed to write folder memory card superblock:\n{}", full_path));
```
but it's a logging message, so doesn't really matter